spec/controllers/restaurants_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+
+describe RestaurantsController, type: :controller do
+  render_views
+
+  let(:restaurant) { FactoryGirl.create(:restaurant) }
+
+  describe "GET" do
+    it "renders the index view" do
```

Comment:
We usually describe 'it' items with a phrase like this:
it "should render the index view"